docs/framework/data/adonet/sql/handling-null-values.md

```diff
@@ -88,7 +88,7 @@ WHERE TerritoryID IN (1, 2, 3)
   
  In addition, the following rules apply for an instance of `DataRow.["columnName"]` null assignments:  
   
-1. The default *default* value is `DbNull.Value` for all except the strongly typed null columns where it is the appropriate strongly typed null value.  
```

Comment:
This one was possibly intentional, given the italicization of the second word. I can see there being a default default value.

Reply:
True, I can't think of a better way of expressing it though if it was intentional 😦